src/mpi/request/waitall.c

```diff
@@ -25,13 +25,6 @@ int MPI_Waitall(int count, MPI_Request array_of_requests[], MPI_Status array_of_
 #endif
 /* -- End Profiling Symbol Block */
 
-/* Define MPICH_MPI_FROM_PMPI if weak symbols are not supported to build
```

Comment:
In the commit message, can you add a little bit more explanation -- why this was needed, where the cache miss happened, and how this patch would solve it?

Reply:
added